keps/sig-node/4540-strict-cpu-reservation/kep.yaml

```diff
@@ -32,7 +31,7 @@ milestone:
 # The following PRR answers are required at alpha release
 # List the feature gate name and the components for which it must be enabled
 feature-gates:
-  - name: "CPUManagerPolicyAlphaOptions"                                                
+  - name: "CPUManagerPolicyBetaOptions"
```

Comment:
I might be mistaken, but I believe we should capture all feature gates used throughout the entire lifecycle of this feature. That would mean specifying both `CPUManagerPolicyAlphaOptions` and `CPUManagerPolicyBetaOptions`. The PRR team can clarify if needed.

Reply:
ok, restored the alpha option.